src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -8,17 +8,9 @@
     package {@view.packageName}
 
     import (
-        "fmt"
-        "runtime"
-        "time"
-
-        "github.com/googleapis/gax-go"
-        "golang.org/x/net/context"
-        "google.golang.org/api/option"
-        "google.golang.org/api/transport"
-        "google.golang.org/grpc"
-        "google.golang.org/grpc/codes"
-        "google.golang.org/grpc/metadata"
+    @join imp : view.imports
+    {@imp}
```

Comment:
The problem here is that Go has a blank line in the middle of imports. The spaces in front of each import is stored in `view.imports` itself. The lack of indent here is so that we don't indent that blank line. I'll comment this behavior.